core/src/main/java/bio/terra/pearl/core/dao/participant/EnrolleeRelationDao.java

```diff
@@ -57,4 +57,42 @@ public void attachTargetEnrollees(List<EnrolleeRelation> relations) {
         }
     }
 
+    /**
+     * This method works like the original findAllByTwoProperties method, but it only returns relations that there
+     * end dates are greater than or equal to the current date.
+     * @param column1Name the name of the first column to filter by
+     * @param column1Value the value of the first column to filter by
+     * @param column2Name the name of the second column to filter by
+     * @param column2Value the value of the second column to filter by
+     * @return a list of EnrolleeRelation objects that match the given properties and have a valid end date.
+     */
+    protected List<EnrolleeRelation> findAllValidByTwoProperties(String column1Name, Object column1Value,
+                                             String column2Name, Object column2Value) {
+        return jdbi.withHandle(handle ->
+                handle.createQuery("select * from " + tableName + " where " + column1Name + " = :column1Value"
+                                + " and " + column2Name + " = :column2Value and end_date >= NOW();")
```

Comment:
do we need to update this clause to  "and (end_date >= NOW() OR end_date is null)" ?